src/protocol.cpp

```diff
@@ -137,9 +137,13 @@ bool CMessageHeader::IsCommandValid() const
 
 ServiceFlags GetDesirableServiceFlags(ServiceFlags services) {
     if ((services & NODE_NETWORK_LIMITED) && g_initial_block_download_completed) {
-        return ServiceFlags(NODE_NETWORK_LIMITED | NODE_WITNESS);
+        // Dogecoin: Disable segwit
+        // return ServiceFlags(NODE_NETWORK_LIMITED | NODE_WITNESS);
```

Comment:
I think this needs to be its own PR so we can revert it individually later, I'll sort that.